### ISSUE TITLE
ElasticSearch7 README instructions don't make sense anymore

### ISSUE THREAD

**Issue by james-deee** [Netflix/conductor-community#55]
The instructions found in the README for elasticsearch7 no longer makes any sense. Can someone update it so that ElasticSearch7 can function with this new setup of using this Library instead of a direct project in Conductor?

https://github.com/Netflix/conductor-community/tree/main/index

The main build.gradle file for example no longer has any mention of ES6 or ES7.

**Comment by spasaria** [comment]
You're right, I hope they update the guide soon. Also in my own installation elastic 7 gives problems and I don't know how to solve.

**Comment by emr17** [comment]
> The instructions found in the README for elasticsearch7 no longer makes any sense. Can someone update it so that ElasticSearch7 can function with this new setup of using this Library instead of a direct project in Conductor?
> 
> https://github.com/Netflix/conductor-community/tree/main/index
> 
> The main build.gradle file for example no longer has any mention of ES6 or ES7.

hi. what are the new instructions?
